### PR TITLE
Update readme for 1.1

### LINKED ISSUE #223: Prepare readme and documentation for 1.1 release

### PR DESCRIPTION
**Pull request by rcrichton**
Closes #223

### CODE REVIEW

**Comment by rcrichton** [comment]
Hey @devcritter could you have a look over the new readme?

File: README.md [hunk 1]
```diff
@@ -9,9 +9,50 @@ This application provides a web application to configure and manage the [OpenHIM
...
+* Importing and exporting OpenHIM server configuration
 
 See the [development roadmap](https://github.com/jembi/openhim-console/wiki/OpenHIM-console-Development-Roadmap) for more details on what is to come!
 
+See [the documentation](https://github.com/jembi/openhim-console/wiki) for more details to get started.
+
+Getting started with the OpenHIM Console
+----------------------------------------
+
+First ensure that you have the OpenHIM-core server up and running. The console communicates with the OpenHIM-core via it's API to pull and display data. See [details on how to get the OpenHIM-core setup](https://github.com/jembi/openhim-core-js/blob/master/README.md).
```
Comment (devcritter) [inline README.md]: type: it's -> its

File: README.md [hunk 2]
```diff
@@ -9,9 +9,50 @@ This application provides a web application to configure and manage the [OpenHIM
...
+wget https://github.com/jembi/openhim-console/releases/download/vX.X.X/openhim-console-vX.X.X.tar.gz
+tar -vxzf openhim-console-vX.X.X.tar.gz --directory /var/www/
+```
+
+Next, and this step is _vital_, you need to configure the console to point to your OpenHIM-core server. Locate `config/default.js` in the folder you extracted the OpenHIM console to and edit it as follows:
+
+```
+{
+  "protocol": "https",
+  "host": "localhost", // change this to the hostname for your OpenHIM-core server (This hostname _MUST_ be publically acessible)
```
Comment (devcritter) [inline README.md]: type: acessible

**Comment by devcritter** [comment]
@rcrichton looks good to me! Just mentioned a couple of typos. Another thing that might be good is to add a link to openhim.org? Maybe right at the top? But other than that :+1:

**Comment by rcrichton** [comment]
Thanks Hannes, good idea.

**Pull request merged**